generate/beat/{{cookiecutter.beat}}/beater/{{cookiecutter.beat}}.go

```diff
@@ -13,42 +13,29 @@ import (
 )
 
 type {{cookiecutter.beat|capitalize}} struct {
-	config config.Config
-	done   chan struct{}
-	client publisher.Client
+	done       chan struct{}
```

Comment:
It could be nice to have the done channel also in under Beat.Done so simple beats could reuse it without having to create or close one themself.